hist/hist/src/TH1.cxx

```diff
@@ -106,7 +106,7 @@ ROOT supports the following histogram types:
       - TH1C : histograms with one byte per channel.   Maximum bin content = 127
       - TH1S : histograms with one short per channel.  Maximum bin content = 32767
       - TH1I : histograms with one int per channel.    Maximum bin content = INT_MAX (\ref intmax "*")
-      - TH1F : histograms with one float per channel.  Maximum precision 7 digits
+      - TH1F : histograms with one float per channel.  Maximum precision 7 digits, maximum bin content = +/-1e7
```

Comment:
We should use here the maximum integer represented exactly by a float 2^24 = 16777216, after that adding 1 will not change the max value.

Reply:
changed.